src/toil/wdl/wdltoil.py

```diff
@@ -1316,95 +1318,6 @@ def add_paths(task_container: TaskContainer, host_paths: Iterable[str]) -> None:
             task_container.input_path_map[host_path] = container_path
             task_container.input_path_map_rev[container_path] = host_path
 
-def import_files(environment: WDLBindings, toil: Toil, path: Optional[List[str]] = None, skip_remote: bool = False) -> WDLBindings:
```

Comment:
I don't think this function is needed? At first glance, our virtualization step should do either the same thing or at least something similar.

Reply:
This function actually is useful as it can import files from URLs given a relative path